automatic/dependencies/tools/chocolateyinstall.ps1

```diff
@@ -22,3 +22,14 @@ $packageArgs = @{
 }
 
 Install-ChocolateyZipPackage @packageArgs
+
+Write-Verbose "Creating start menu entry"

```

Comment:
I would prefer a package parameter whether those shortcuts should be created or not.
When using e.g. keystroke launchers those are not necessary...